Makefile

```diff
@@ -223,6 +223,10 @@ enable-git-hooks:
 docs: ## Create documentation under docs/build/html
 	./docs/source/build-docs.sh
 
+.PHONY: docs-live
```

Comment:
nit: the convention in this makefile is to use underscores `_` rather than dashes `-`.